guide/payments/send.md

```diff
@@ -39,169 +103,255 @@ layout = "full-width"
 
-- Allow senders to switch which denomination they are inputting the amount with
-- State the total balance available (subtracting the required fee)
-- Allow selecting max amount or fractions for easier fund management
-- Payment links and QR Codes can contain an amount. When they do, your application should populate the amount field automatically
-- Indicate if an amount entered is more than the available balance
+**On-chain fees**
 
-## Setting the transaction fee
+This fee is dependent on how many other transactions are currently waiting to be processed on the base layer as a whole. The [average fee](https://ycharts.com/indicators/bitcoin_average_transaction_fee) in January 2021 was $0.63, and $28.60 in April 2021.
+
+</div>
+
+## Review & approval
 
 <div class="center" markdown="1">
+
 {% include image.html
-   image = "/assets/images/guide/payments/send/select-fee-rate.png"
-   retina = "/assets/images/guide/payments/send/select-fee-rate@2x.png"
-   alt-text = "Fee selection component with urgent, normal, and not urgent options"
-   width = 400
-   height = 470
-   caption = "G = Haitian Gourde, the local currency of Haiti."
-   layout = "float-right-desktop"
+   image = "/assets/images/guide/payments/send/confirm.png"
+   retina = "/assets/images/guide/payments/send/confirm@2x.png"
+   alt-text = "Example image"
+   caption = "A compact summary to confirm the information is accurate."
+   width = 250
+   height = 541
+   layout = "float-right-desktop -background -shadow"
 %}
 
-The application can automatically estimate a fee and set it for the sender. This would typically prioritize the transaction to be included in a block as soon as possible. Since the fee rate may vary if the network is busy, you can give senders more fine-grained fee controls so they can choose to optimize for faster confirmation or lower fees.
+Particularly with payments for larger amounts, it is good practice to allow users to review and confirm the information before it is sent.
 
-When allowing the sender to set fees, it is essential to communicate the estimated cost in their preferred currency and the estimated time until first confirmation. Be mindful that the way Bitcoin fees get calculated may not map to the sender's traditional financial experience. If you allow adjustment of fees, consider providing details on how it gets calculated.
+Payments initiated via invoices which don't require any additional input by the user can avoid this review step, as the whole user interaction is one of review and confirmation.
+
+Wallets that offer features for [pending limits]({{ '/guide/onboarding/protecting-a-wallet/#wallet-limits' | relative_url }})s or support [multi-key]({{ '/guide/private-key-management/multi-key/' | relative_url }}) schemes for additional security may ask users to go through additional confirmation steps here.
```

Comment:
```suggestion
Wallets that offer features for [payment limits]({{ '/guide/onboarding/protecting-a-wallet/#wallet-limits' | relative_url }}) or support [multi-key]({{ '/guide/private-key-management/multi-key/' | relative_url }}) schemes for additional security may ask users to go through additional confirmation steps here.
```
Should this be payment limits? I don't think we should mention multi-key here, the wallet product this section is covering should be LN focused which would not encompass a multi-key element.

Reply:
```suggestion
Wallets that offer features for [spending limits]({{ '/guide/onboarding/protecting-a-wallet/#wallet-limits' | relative_url }}) or support [multi-key]({{ '/guide/private-key-management/multi-key/' | relative_url }}) schemes for additional security may ask users to go through additional confirmation steps here.
```

Reply:
@Bosch-0 it was meant to be spending limits.